Makefile

```diff
@@ -39,6 +51,18 @@ start: ## Start the development environment (use Docker)
 stop: ## Stop FreshRSS container if any
 	docker stop freshrss-dev
 
+###########
+## Tests ##
+###########
+bin/phpunit: ## Install PHPUnit for test purposes
```

Comment:
@marienfressinaud Here you assumed that the user use a Linux machine. I am ok with that but it could be nice to have some validation mecanism to notify the user.

Reply:
You can also run that in the docker container if it exists